tutorials/tutorial2.ipynb

```diff
@@ -12,7 +12,7 @@
     "\n",
     "We selected 2400 NK cells from this dataset - this is our ‘clean’, high-coverage dataset. We then randomly sampled 50 of these 2400 NK cells. Here's what the ATAC-seq signal from 50 cells and 2400 cells looks like, for a region on chromosome 10:\n",
```

Comment:
For line 11 - the section name is "AtacWorks generalizes to diverse applications"

Reply:
done.